templates/minimal/README.md

```diff
@@ -1,13 +1,93 @@
-# Minimal Template
+<div align="center">
 
-This is a minimal template for creating a blockchain using the Polkadot SDK.
+# Polkadot SDK's Minimal Template
 
-# Docs
```

Comment:
So the question is what is the minimal template? 

The main difference between the minimal template and others is that it has the least number of components to run. 

In the runtime side, this template only has a handful of pallets, which you can list. 

On the node side, this template has no consensus and can run both with custom block time via CLI. see `enum Consensus` in the `node` folder.

In general, for palying around with a single node, this template is best.

Reply:
I felt like some more introduction of what you're looking at is missing.

I added something based on this comment.